server/chat-plugins/youtube.ts

```diff
@@ -267,22 +262,28 @@ export const commands: ChatCommands = {
 			this.privateModAction(`(${user.name} updated channel ${id}'s username to ${name}.)`);
 			return FS(STORAGE_PATH).writeUpdate(() => JSON.stringify(channelData));
 		},
+		interval: 'repeat',
 		repeat(target, room, user) {
 			if (room.roomid !== 'youtube') return this.errorReply(`This command can only be used in the YouTube room.`);
 			if (!this.can('declare', null, room)) return false;
-			if (!target || isNaN(parseInt(target))) return this.errorReply(`Specify a number (in minutes) for the interval.`);
+			if (!target) return this.sendReply(`Interval is currently set to ${Chat.toDurationString(YouTube.timer)}.`);
+			if (Object.keys(channelData).length < 1) return this.errorReply(`No channels in the database.`);
+			if (isNaN(parseInt(target))) return this.errorReply(`Specify a number (in minutes) for the interval.`);
 			let interval = Number(target);
 			if (interval < 10) return this.errorReply(`${interval} is too low - set it above 10 minutes.`);
 			interval = interval * 60 * 1000;
+			const res = await YouTube.randChannel();
+			// no channels 
+			if (!res) return this.errorReply(`Error in getting channel data.`);
+			YouTube.timer = interval;
```

Comment:
What happens if `YouTube.timer` already exists?

Reply:
it should be overwritten, i believe.

Reply:
Oh, wait, `YouTube.timer` and `YouTube.interval` are just confusingly named.

Maybe `YouTube.interval` and `YouTube.intervalTime` would be clearer?